src/MotoROS_PlatformLib.h

```diff
@@ -55,8 +55,23 @@ extern ULONG tickGet();
 #define SO_BROADCAST    0x0020
 extern STATUS Ros_setsockopt(int s, int level, int optname, char* optval, int optlen);
 
-
+#if defined (DX100) || defined (FS100)
+// VxWorks 5.5/6.8
 extern int localtime_r(const time_t* timer, struct tm* timeBuffer);
+#else
+// >= VxWorks 6.9
+struct tm* localtime_r(const time_t* timep, struct tm* result);
+#endif
+
+
+// Attempts to determine whether the specific link is UP (ie: cable is connected).
+// IFF return value is 'OK', 'is_up' will reflect link state. In all other cases,
+// 'is_up' may be left uninitialised.
+#define ROS_USERLAN_STATE_INVALID_IF_NO -10
+#define ROS_USERLAN_STATE_NO_SUCH_IF -11
+#define ROS_USERLAN_STATE_GET_STATE_ERROR -12
+#define ROS_USERLAN_STATE_GENERIC_FAILURE -13
+extern STATUS Ros_UserLan_IsLinkUp(USHORT if_no, BOOL* const is_up /*out*/);
```

Comment:
These lines should not be part of this PR. They are already part of #49.

Reply:
I just gave SejalBehere the latest version of the file which included the update for `localtime_r`.

Since the history of this file is being maintained in another repo, I'm not sure how you want to handle it in this repo. Do you just want the `localtime_r` change in here? What about the `MOTOROS_PLATFORM_LIB_VERSION`?

Reply:
I'd suggest to remove everything *not* related to `localtime_r(..)`. After merging this PR, I'll rebase #49 and sort out any conflicts.

It's a bit awkward with the binary involved, but we'll figure it out.